data/tools/converters.yml

```diff
@@ -384,3 +388,279 @@
             url: https://hub.docker.com/r/mercureimaging/mercure-dcm2bids
         GUI: true
         license: GPL-3.0
+
+
+# EEG, MEG, iEEG converters
```

Comment:
I would remove those comments / sections and sort all the converters by alphabetical order of their name: less ambiguous for organizing the entries